tests/ingresstests/ingressclass_test.go

```diff
@@ -191,6 +193,70 @@ func TestDefaultIngressClassChange(t *testing.T) {
 	VerifyPoolDeletionFromVsNode(g, modelName)
 }
 
+func TestIngressWithNonAVILBIngressClass(t *testing.T) {
+	// create ingress with ingressClass avi-lb
```

Comment:
Comments are wrong. Here we are creating first with wrong ingressclass and then update with right one,

Reply:
I have corrected the comment. Thanks